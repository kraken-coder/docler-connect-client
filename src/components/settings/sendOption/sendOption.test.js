import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import React from 'react'
import SendOption from './sendOption'

test('match snapshot', () => {
  const Wrapper = shallow(<SendOption />)
  expect(toJSON(Wrapper)).toMatchSnapshot()
})
