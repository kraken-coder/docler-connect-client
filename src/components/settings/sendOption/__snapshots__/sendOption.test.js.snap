// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot 1`] = `
<div
  className="settings-items"
>
  <p
    className="settings-heading"
  >
    Send Message on Ctrl + enter
  </p>
  <div
    className="inputGroup"
  >
    <input
      id="on"
      name="sendOption"
      onChange={[Function]}
      type="radio"
    />
    <label
      htmlFor="on"
    >
      Yes
    </label>
  </div>
  <div
    className="inputGroup"
  >
    <input
      id="off"
      name="sendOption"
      onChange={[Function]}
      type="radio"
    />
    <label
      htmlFor="off"
    >
      No
    </label>
  </div>
</div>
`;
